src/frontend/css/plugins/color.js

```diff
@@ -0,0 +1,69 @@
+const colorFunctions = require('less/lib/less/functions/color').default;
+
+const SUPPORTED_COLOR_FUNCTIONS = {
+  LIGHTEN: 'lighten',
+  DARKEN: 'darken',
+  FADE: 'fade'
+};
+
+// TODO (esn-frontend-common-libs#51): Write tests for this plugin
+function colorFunctionInterceptor(color, amount, method, functionName) {
+  if (!Object.values(SUPPORTED_COLOR_FUNCTIONS).includes(functionName)) {
+    throw new Error(`The LESS's color function '${functionName}' is not supported with CSS Variables`);
+  }
+
+  // If the "color" argument is a known, concrete color, not a CSS Variable, we'll just use the default LESS function.
+  if (color.toHSL) return colorFunctions[functionName](color, amount, method);
```

Comment:
Not sure about everyone else, but I really don't like this kind of if statement (without braces).
There is even an ESLint rule about it: https://eslint.org/docs/rules/curly.

What do you think @tuanlc ?
Also, there are some other places in @huy-ta 's PR that has this style. If it need to be fixed, please fix other places as well.

Reply:
I think for short statements, they can sit on the same line. Tbh it's just a matter of preference. There are many places in our code (especially in our backend code) use this style.
ESLint is not a dictator though. In fact, it allows us to abide by the rules that we think are useful and drop the rules that are not. For example, even though https://eslint.org/docs/rules/comma-dangle exists, not everybody follows it.

Reply:
For me, I'm ok without braces in this case.